### ISSUE TITLE
Compile v4l2rtspserver

### ISSUE THREAD

**Issue by LeVuHoang** [samtap/fang-hacks#386]
Hi all,

We have so many issue with rtsp service (snx_rtsp_server) because it's an old version.

I used Dafang hacks with new Xiaofang camera and it's faultlessly. Dafang hacks use v4l2rtspserver-master service.

Is there anyone able to compile v4l2rtspserver for this old camera or tell me how to do it?

https://github.com/mpromonet/v4l2rtspserver

**Comment by LeVuHoang** [comment]
I'm following this instruction and hopefully I can compile a new version
https://github.com/Dafang-Hacks/Main

**Comment by arifwib** [comment]
hi, @LeVuHoang any progress on compiling v4l2rtspserver and use it with samtap fang-hacks ?

**Comment by 350d** [comment]
@LeVuHoang Can't wait for update!

**Comment by LeVuHoang** [comment]
Hi guys,

I use
`snx_snapshot -m -q 100 -n 24 -e 0`

and it works perfectly. No crash.

**Comment by 350d** [comment]
@LeVuHoang wrong thread?

**Comment by LeVuHoang** [comment]
> @LeVuHoang wrong thread?
No, nobody tells me a way to compile so I have to use snx_snapshot

**Comment by 350d** [comment]
@LeVuHoang for video streaming or for image snapshots?

**Comment by LeVuHoang** [comment]
I use Milestone Xprotect to capture this as snapshot but you can still see it as video streaming
